CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Rojo Changelog
 
 ## Unreleased Changes
```

Comment:
We may also want to mention `Content` properties, and their possible impact on existing Rojo projects.